### ISSUE TITLE
Fast deploy

### ISSUE THREAD

**Issue by Fewiel** [Fewiel/EnvironmentServer#94]
- Automatically create templates for new versions
- Allow choosing fast deploy option or normal installation
- Fast deploy should take 5 seconds

Depends on #88 Templates

**Comment by Fewiel** [comment]
Not needed so much - Its already very fast to create a new environment.

**Comment by Fewiel** [comment]
Won't do
